Fixed some syntactical errors in zener.lib

nexx512 opened the pull request:
zener.lib didn't work with ngspice.
I tried to fix the syntactical errors in the file.
For some models I had to add a `.ENDS` statement manually. These models seem to be missing some information. Therefore I added some comments that these models are probably currupt.

aster94 commented:
Thanks for the PR!
ok for `zener.lib`, but what is `DIODE2.lib` maybe it is an error and shouldn't be into the PR?

nexx512 commented:
Sorry, `diode2.lib` was accidentally commited. It was in the repo before. I just changed file permissions and renamed it from uppercase to lowercase for asthetic reasons. Is that an issuer? If so I will fix that and update the PR.

aster94 commented:
yes, sorry but github think it's a new file, can you remove it from the PR?

nexx512 commented:
ok, that's strange. When I look at the changes of the PR in github it shows the change of file permissions. But I will see what I can do.
![grafik](https://user-images.githubusercontent.com/24672655/146262029-c074a3a0-5218-447a-bd3e-7d477c146f50.png)

nexx512 commented:
Fixed it.

aster94 commented:
Yes, it's perfect
